packages/ui-react/src/components/Header/Header.tsx

```diff
@@ -195,6 +202,21 @@ const Header: React.FC<Props> = ({
     );
   };
 
+  const renderNav = () => {
+    if (navItems.length === 0) {
+      return children;
```

Comment:
Isn't `children` always empty now? I think you want to return `null`.

Reply:
Yes. The value is `undefined` now, which is returned. This is identical to returning `null`. So I suggest keeping it as-is. We could fix it by giving a default value to `children` when restructuring the props, but it is not needed.